wavefront/server/apps/floconsole/floconsole/controllers/app_user_controller.py

```
@@ -0,0 +1,233 @@
+            filtered_users = [u for u in users if u.id in user_ids]
+            users_data = [user.to_dict() for user in filtered_users]
+        else:
+            users_data = []
+
+        logger.info(f'Retrieved {len(users_data)} users for app {app_id}')
+
+        return JSONResponse(
+            status_code=status.HTTP_200_OK,
+            content=response_formatter.buildSuccessResponse({'users': users_data}),
+        )
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)